tests/unit/amo/components/TestReportAbuseButton.js

```diff
@@ -57,9 +57,9 @@ describe(__filename, () => {
 
     expect(root.find('.ReportAbuseButton')).toHaveLength(1);
     expect(root.find('.ReportAbuseButton-show-more').prop('children'))
-      .toEqual('Report this add-on for abuse');
+      .toEqual('Report an issue with this add-on');
```

Comment:
It might be worth adding a comment explaining why this text contains the word "abuse" (linking to https://github.com/mozilla/addons-frontend/issues/4025#issuecomment-349103373, perhaps) so we don't try to improve the wording again in the future without realising the context.